### PR TITLE
Adds documentation

### PR DESCRIPTION
**Pull request by arcanis**
This PR adds the documentation relevant to the Node [project requirements](https://github.com/nodejs/admin/blob/master/transfer-repo-into-the-org.md#step-1-get-the-repository-ready) ✨

cc @mmarchini / https://github.com/nodejs/admin/issues/558#issuecomment-701009551

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Various tidbits about Corepack's design are explained in more details in [DESIGN
 
 ## License (MIT)
 
-> **Copyright © 2019 Maël Nison**
+> **Copyright © 2019 Corepack contributors**
```
Comment (jasnell) [inline README.md:115]: Likely best just to remove the year

File: LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+**Copyright © 2019 Corepack contributors**
```
Comment (jasnell) [inline LICENSE.md:1]:
```suggestion
**Copyright © Corepack contributors**
```

**Review by jasnell — APPROVED** [review]
Minor nit but LGTM

**Review by mmarchini — COMMENTED** [review]
I'm late but lgtm

**Pull request merged**